### DISCUSSION TITLE
DietPi-Software | Kavita

### DISCUSSION THREAD

**Discussion by m1rr0r5h4d35** [MichaIng/DietPi#5449] [Software Request] [answered]
#### Is your feature request related to a problem? Please describe:

- ...
I think a better ebook solution than Ubooquity is needed
#### Describe the solution you'd like:

- ...
It would be nice to have Kavita as an alternative to Ubooquity https://www.kavitareader.com/

#### Describe alternatives you've considered:

- ...
Calibre Web is nice, but requires Calibre Desktop app as well. While I am not opposed to this, I think most folks are looking for a "Plex for Ebooks" type of solution and Kavita is trying to work toward that.
#### Additional context

- ...

**Comment by MichaIng** [comment]
Just for completeness, Komga as alternative ebooks server is supported already: https://dietpi.com/docs/software/media/#komga

**Comment by MichaIng** [comment, marked as answer]
PR up: #6610

**Comment by m1rr0r5h4d35** [comment]
My apologies for the delay. Thank you for the suggestion, I had not heard of Komga before, so I gave it a spin and it works nicely. It seems like there are others who liked the Kavita suggestion, and I see work has been done to integrate it into the DietPi ecosystem. I am glad others found the suggestion useful. Thank you for your time and efforts. I appreciate all that you do.
